plugins/modules/packaging/os/rhn_register.py

```diff
@@ -186,6 +199,9 @@ def systemid(self):
             xml_data = fd.read()
             fd.close()
 
+            # These imports are here to avoid import issues where XML packages are missing
+            # pylint: disable=bad-option-value,import-outside-toplevel
```

Comment:
This is not needed for pylint in CI. I think it's better to not add it here.

Reply:
Does the CI run with a less restrictive pylint level than the sanity checks?
Without these pylint disables I get this:
```
$ ansible-test sanity --test pylint plugins/modules/packaging/os/rhn_register.py
Running sanity test 'pylint' with Python 3.9
ERROR: Found 3 pylint issue(s) which need to be resolved:
ERROR: plugins/modules/packaging/os/rhn_register.py:206:20: import-outside-toplevel: Import outside toplevel (libxml2)
ERROR: plugins/modules/packaging/os/rhn_register.py:218:20: import-outside-toplevel: Import outside toplevel (lxml.etree)
ERROR: plugins/modules/packaging/os/rhn_register.py:222:20: raise-missing-from: Consider explicitly re-raising using the 'from' keyword
See documentation for help: https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/pylint.html
ERROR: The 1 sanity test(s) listed below (out of 1) failed. See error output above for details.
pylint
```

Reply:
If you use the default docker container, you can replicate the behavior in CI: `ansible-test sanity --docker default -v --test pylint plugins/modules/packaging/os/rhn_register.py`

If you don't use `--docker` or `--venv`, you are using your local `pylint`, which is another (probably newer) version - and thus reports potentially other things.

Reply:
Indeed it was, thanks! I'll remove these now. Do I need to squash my commits or will that be done on PR?

Reply:
Squashing is not needed, we always squash on merging. (Squashing before only makes sense if you have an excessive amount of commits, for example the bot will start complaining when you have 50 in the PR ;-) )